File: examples/python/sql_query_example.py
import qoery

configuration = qoery.Configuration()
configuration.api_key['ApiKeyAuth'] = "your-api-key"

with qoery.ApiClient(configuration) as api_client:
    api = qoery.QueriesApi(api_client)
    request = qoery.QuerySqlPostRequest(query="SELECT * FROM series LIMIT 10")
    response = api.query_sql_post(request).to_dict()
    

File: examples/js/sql_query_example.js
var Qoery = require('qoery');
var defaultClient = Qoery.ApiClient.instance;
defaultClient.authentications['ApiKeyAuth'].apiKey = "your-api-key";

var api = new Qoery.QueriesApi();
var req = new Qoery.QuerySqlPostRequest();
req.query = "SELECT * FROM series LIMIT 10";

api.querySqlPost(req, function(err, data) {
  if (err) { console.error(err); return; }
  console.log(data.sql_query);
});
